docs/source/usage_guides/mps.mdx

```diff
@@ -0,0 +1,100 @@
+specific language governing permissions and limitations under the License.
+-->
+
+# Accelerated PyTorch Training on Mac
+
+With PyTorch v1.12 release, developers and researchers can take advantage of Apple silicon GPUs for significantly faster model training. 
+This unlocks the ability to perform machine learning workflows like prototyping and fine-tuning locally, right on Mac.
+Apple's Metal Performance Shaders (MPS) as a backend for PyTorch enables this and can be used via the new `"mps"` device. 
+This will map computational graphs and primitives on the MPS Graph framework and tuned kernels provided by MPS.
+For more information please refer official documents [Introducing Accelerated PyTorch Training on Mac](https://pytorch.org/blog/introducing-accelerated-pytorch-training-on-mac/)
+and [MPS BACKEND](https://pytorch.org/docs/stable/notes/mps.html).
+
+### Benefits of Training and Inference using Apple M1 Chips
+
+1. Enables users to train larger networks or batch sizes locally
+2. Reduces data retrieval latency and provides the GPU with direct access to the full memory store due to unified memory architecture. 
+Therefore, improving end-to-end performance.
+3. Reduces costs associated with cloud-based development or the need for additional local GPUs.
+
+### Pre-requisites: Installing torch with mps support
+
+Following this great medium article [GPU-Acceleration Comes to PyTorch on M1 Macs](https://medium.com/towards-data-science/gpu-acceleration-comes-to-pytorch-on-m1-macs-195c399efcc1),
+below are the steps to make sure torch is installed with `mps` support.
+
+```
+# installing torch with m1 support on mac
+# install latest python from https://www.python.org/downloads/release/python-3106/
+# check the platform
+import platform
+platform.platform()
+'macOS-12.5-arm64-arm-64bit' 
+# This is compatible as the macOS version is above 12.3 and it is the ARM64 native version of Python
+# install torch 1.12 via the below command
+# pip3 install torch torchvision torchaudio
+# test the `mps` device support
+import torch
+torch.has_mps
+True
+a = torch.Tensor([10,11])
+a.to("mps")
+/Users/mac/ml/lib/python3.10/site-packages/torch/_tensor_str.py:103: UserWarning: The operator 'aten::bitwise_and.Tensor_out' is not currently supported on the MPS backend and will fall back to run on the CPU. This may have performance implications. (Triggered internally at  /Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/mps/MPSFallback.mm:11.)
+  nonzero_finite_vals = torch.masked_select(tensor_view, torch.isfinite(tensor_view) & tensor_view.ne(0))
+tensor([10.0000, 11.0000], device='mps:0')
+```
+
+## How it works out of the box
+
+On your machine(s) just run:
+
+```bash
+accelerate config
+```
+
+and answer the questions asked, specifically choose `MPS` for the query `Which type of machine are you using?` . 
+This will generate a config file that will be used automatically to properly set the
+default options when doing
```

Comment:
I can't make a suggestion for this one, but I recommend something like this:

and answer the questions asked, specifically choose `MPS` for the query:
```
Which type of machine are you using?
```
This will generate a config file that will be used automatically to properly set the
default options when doing